product-service/src/utils/errors.ts

```diff
@@ -0,0 +1,9 @@
+export const serverError = {
+  statusCode: 500,
+  message: 'Internal server error',
+};
+
+export const clientError = {
+  statusCode: 400,
```

Comment:
Just FYI. I'm pretty sure there was an error when was requested to show 400 code. Usually people  [use 422 code](https://developer.mozilla.org/ru/docs/Web/HTTP/Status/422) that tells the user that there is a format error with meaningful message. (It's worth mentioning that from my perspective it's hard to understand what is wrong when I send invalid data to create new car in the database, cause the message is too generous)